brew-cask: refresh page (#8925)
tldr-pages/tldr: pages/common/brew-cask.md

> # brew cask

Reviewer: It's a little confusing to have `brew cask` as command name, but then document `brew --cask`. I think we should rather completely replace `brew cask` with `brew --cask` and add a note like ```This command previously was called as `brew cask`, but has been deprecated in favor of `brew --cask` ```.

Author: Agreed, I will update it tomorrow. Bye

> Package manager for macOS applications distributed as binaries.

> > This command has been depreciated from upstream and it is replaced by `brew --cask` flag.

Reviewer 2: ```suggestion
> This subcommand has been depreciated and replaced by the `--cask` flag.
```
